teaching_reproducibility.Rmd

```diff
@@ -367,61 +420,128 @@ especially those who are also new to the command line in general.
-and outside of the classroom in their own work. 
-And so we actually want to change their habits or behaviors. 
-We think it's quite important to realize that it's not
-just understanding an algorithm, with reproducibility,
-it's it's understanding the concepts behind something like version control, 
-knowing how to use those things and then
-once you leave the classroom wanting to and being able to use these things.
-Essentially, we want to get students to the point where they do these things
-out of habit, and they don't really have to think to hard about how to do it,
-nor think about other ways to do it. 
-
-One thing thing that can be done in the classroom to embed lots of practice 
-in a complementary ways with live demos is to intentionally pause during the
-demos and say "okay students your turn, do what i just did", and then give 
-them time for the practice of typing it into the keyboard or clicking their
-mouse around the graphical user interface. 
-Another way this can be done is through lots of low stakes assessments
-comprised of small or short problems. In data science, this works well in 
-literate code documents that have automated tests in them to 
-provide automated feedback. Another way lots practice can be provided is 
-through the choice of learning technologies and platforms that are used in the 
-course. Using authentic data science reproducibility tools as part of the 
-course learning technology mechanics (homework submission, grading, etc)
-provides many opportunities for authentic practice.
+doesn't require repetition.
+However, that is not how we learn most of the topics taught at school,
+which tend to be less emotionally salient. 
+Such topics are instead better learned through repetitive spaced practice,
+where information is committed to long-term memory
+by revisiting and repeating it multiple times 
+with break in between.
+
+When we teach reproducibility topics,
+we want students to do more than commit information to long term behavior.
+We actually want them to act on new information and induce a change in behavior
+by putting their knowledge into practice,
+both in the classroom and in their own work.
+Importantly and in contrast to many other data science topics,
+students often already have behavioral patterns in place for how they organize and name their files, or how they collaborate with their colleagues.
+Although these are not reproducible practices,
+they fill many of the same niches in a learner's workflow,
+e.g. we want students to replace naming documents `draft-final-version-3.md`
+with using git and GitHub for version control
+and substitute collaboration via forks and PR on GitHub
+for their current habits of using working together on Google docs.
+Therefore,
+teaching reproducibility ought to go one step beyond promoting effective retention of information
+and focus on habitual and behavioral changes,
+which introduces additional challenges and opportunities.
```

Comment:
<!--TODO develop on which challenges and more or are the following paragrahs enough?? -->